tests/migration/namespace.go

```diff
@@ -302,6 +303,146 @@ var _ = Describe(SIG("Live Migration across namespaces", decorators.RequiresDece
+
+		addDVVolume := func(name, namespace, volumeName, claimName string, bus v1.DiskBus) {
+			opts := &v1.AddVolumeOptions{
+				Name: volumeName,
+				Disk: &v1.Disk{
+					DiskDevice: v1.DiskDevice{
+						Disk: &v1.DiskTarget{
+							Bus: bus,
+						},
+					},
+					Serial: volumeName,
+				},
+				VolumeSource: &v1.HotplugVolumeSource{
+					DataVolume: &v1.DataVolumeSource{
+						Name: claimName,
+					},
+				},
+			}
+
+			Eventually(func() error {
+				return virtClient.VirtualMachine(namespace).AddVolume(context.Background(), name, opts)
+			}, 3*time.Second, 1*time.Second).Should(Succeed())
+		}
+
+		It("should live migrate a container disk vm, with an additional hotpluggedPVC mounted, should stay mounted after migration", decorators.RequiresRWXBlock, func() {
+			sc, exists := libstorage.GetRWXBlockStorageClass()
+			if !exists {
+				Fail("Fail test when RWXBlock storage class is not present")
+			}
+			migrationID := fmt.Sprintf("mig-%s", rand.String(5))
+			hotpluggedDV := createDVBlock(fmt.Sprintf("dv-%s", migrationID), testsuite.NamespaceTestDefault, sc)
+
+			sourceVMI := libvmifact.NewCirros(
+				libvmi.WithNamespace(testsuite.NamespaceTestDefault),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithCPURequest("1"), libvmi.WithMemoryRequest("128Mi"),
+				libvmi.WithCPULimit("1"), libvmi.WithMemoryLimit("128Mi"),
+				libvmi.WithAnnotation("kubevirt.io/libvirt-log-filters", "3:remote 4:event 3:util.json 3:util.object 3:util.dbus 3:util.netlink 3:node_device 3:rpc 3:access 1:*"),
+			)
+
+			sourceVM := createAndStartVMFromVMISpec(sourceVMI)
+			By("Adding volume to running VM")
+			volumeName := "testvolume"
+			addDVVolume(sourceVM.Name, sourceVM.Namespace, volumeName, hotpluggedDV.Name, virtv1.DiskBusSCSI)
+
+			By("Verifying the volume and disk are in the VMI")
+			Eventually(func() virtv1.VolumePhase {
+				sourceVMI, err = virtClient.VirtualMachineInstance(sourceVMI.Namespace).Get(context.Background(), sourceVMI.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				for _, v := range sourceVMI.Status.VolumeStatus {
+					if v.Name == volumeName {
+						return v.Phase
+					}
+				}
+				return ""
+			}).WithTimeout(time.Minute).WithPolling(2 * time.Second).Should(Equal(virtv1.VolumeReady))
+			sourceVMI, err = virtClient.VirtualMachineInstance(sourceVMI.Namespace).Get(context.Background(), sourceVMI.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			deviceName := ""
+			for _, v := range sourceVMI.Status.VolumeStatus {
+				if v.Name == volumeName {
+					deviceName = v.Target
+					break
+				}
+			}
+
+			By("Writing data to extra disk")
+			Expect(console.LoginToCirros(sourceVMI)).To(Succeed())
+			// I am aware I should not use the device name since it is not guaranteed to be the same as the one in the VMI
+			// I should be using the serial number, but not sure how to access that in cirros.
```

Comment:
Did it also break with alpine? or was this test coupled to cirros? this came up in another PR https://github.com/kubevirt/kubevirt/pull/15863 and I'm pretty sure both alpine variations support this